src/runway/commands/modules_command.py

```diff
@@ -354,7 +352,7 @@ def run(self, deployments=None, command='plan'):  # noqa pylint: disable=too-man
         if deployments is None:
             deployments = self.runway_config['deployments']
         context = Context(env_name=get_env(self.env_root,
-                                           self.runway_config.get('ignore_git_branch', False)),
+                                           self.runway_config.get('ignore-git-branch', False)),
```

Comment:
With the `Config` class, all possible options have a value so we don't have to use `.get` here and instead use the attribute directly.

```suggestion
                                           self.runway_config.ignore_git_branch),
```

Also, since this is `.get`ing a class attribute, it does not have hyphens and will need to be reverted to underscores.